test/helpers.js

```diff
@@ -1105,3 +1105,34 @@ export async function expectErrorThrown(page, cb) {
     await Promise.all([hitErrorBoundary, cb()]);
     assert(hitErrorBoundary);
 }
+
+export async function createOAuth2Client(input) {
+    const resp = await fetch(`${TEST_APPLICATION_SERVER_BASE_URL}/test/create-oauth2-client`, {
+        method: "POST",
+        headers: { "content-type": "application/json" },
+        body: JSON.stringify(input),
+    });
+    return await resp.json();
+}
+
+export async function setOAuth2ClientIdInStorage(page, clientId) {
```

Comment:
Please add a note on why we are storing this in localstorage (I suspect it's because we need it in the callback)

Reply:
Yes, I have added a comment.